keyboards/bakeneko60/bakeneko60.h

```diff
@@ -0,0 +1,61 @@
+/* Copyright 2020 Koichi Katano
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_all( \
```

Comment:
This layout matches the [`60_ansi_split_bs_rshift` Community Layout](https://github.com/qmk/qmk_firmware/tree/4701001524276638992d1ae92b638ca68fd0c410/layouts/default/60_ansi_split_bs_rshift).

```suggestion
#define LAYOUT_60_ansi_split_bs_rshift( \
```